data/json/items/gun/exodii.json

```diff
@@ -225,5 +225,43 @@
         "description": "The most popular gun to use the 12.3ln cartridge was, of course, the PA md. 71.  Its predecessor, the md. 68, was viewed by many as a sort of failure: although it was reliable and powerful, it was too heavy to be used as a good infantry weapon, and not really heavy enough to be a good support gun.  Enough were made, though, that during the zombie apocalypse, it gained a great deal of resurgent popularity as a light emplacement gun that used readily available ammunition.  It perfectly served the purposes of the Exodii, who had far less concern about its unwieldiness.  This one has been modified for use in vehicle turrets on Earth."
       }
     ]
+  },
+  {
+    "id": "ree_33",
+    "copy-from": "rifle_semi",
+    "looks_like": "m2browning",
+    "type": "GUN",
+    "name": { "str": "exodii naval rifle" },
+    "description": "Originally designed for naval defense of some small island country, this gun served Exodii as long as they existed, being one of its reliable tools in later stages of world decay.  Cheap ammunition, a simple mechanism, and immense power output would make it a great weapon for any survivor who is able to carry nearly a centner of its weight and is in need to penetrate a small bunker.  Unless you have a body, compatible with Exodii Standard Combat Bodyframe, you can use it only with a heavy-duty tripod, and only while prone.  Due to the size of a top-mounted magazine, user need to use a scope mounted on the side of a gun.",
```

Comment:
```suggestion
    "description": "Originally designed for naval defense of some small island country, this gun served Exodii as long as they existed, being one of its reliable tools in later stages of world decay.  Cheap ammunition, a simple mechanism, and immense power output would make it a great weapon for any survivor who is able to carry nearly a centner of its weight and is in need to penetrate a small bunker.  Unless your body is compatible with Exodii Standard Combat Bodyframe, you can only use it with a heavy-duty tripod, and only while prone.  Due to the size of a top-mounted magazine, user needs to use a scope mounted on the side of a gun.",
```

Reply:
"Originally designed for naval defense of `a` small island country, this gun `has` served `the` Exodii `for` as long as they `have` existed, being one of `their` reliable tools in `the` later stages of world decay.  Cheap ammunition, a simple mechanism, and immense power output would make it a great weapon for any survivor who is able to carry nearly a `???` of its weight and is in need `of` penetrat`ing` a small bunker.  Unless your body is compatible with `the` Exodii Standard Combat Bodyframe, you can only use it with a heavy-duty tripod, and only while prone.  Due to the size of `the` top-mounted magazine, `the` user `has` to use a scope mounted on the side of `the` gun."

I have no idea what a "centner" is supposed to be. The text is in English, so the units should be recognizable as well (and please don't try "stones").